app/lib/pdf_filler/f13614c_pdf.rb

```diff
@@ -77,6 +77,8 @@ def hash_for_pdf
         "form1[0].page1[0].writtenCommunicationLanguage[0].otherLanguageNo[0]" => @intake.written_language_preference_english? ? '1' : nil,
         "form1[0].page1[0].writtenCommunicationLanguage[0].otherLanguageYou[0]" => @intake.written_language_preference_english? ? nil : '1',
         "form1[0].page1[0].writtenCommunicationLanguage[0].whatLanguage[0]" => @intake.written_language_preference_english? ? nil : @intake.preferred_written_language_string
+      # TODO: This doesn't exist for spouse and it doesn't seem like we collect it anyways. It is part of the ticket though
```

Comment:
just moved this up so it would be closer to the related fields, I think we can remove this though. will confirm with ryan